### DISCUSSION TITLE
Allow the ➕ emoji as issue (comment) reactions

### DISCUSSION THREAD

**Discussion by JosXa** [community/community#52328] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I frequently see issue sections being flooded with "I have the same issue" or "me too" comments when people observe the same problem as the OP. I think a simple solution to this could be to allow using the plus ➕ reaction emoji (in addition to thumbsup 👍, which doesn't quite have the same meaning). Whoever wants to add their voice can do so using two clicks and make it easier for large repo owners to gauge impact. It would look like +1, then +2, +3, etc.

I don't know how this would tie in with the upvote feature we have in Discussions or whether there are plans to add those to issues aswell...

**Comment by kenwang-coursera** [comment]
+1

A thumbs up is _not_ a "+". A thumbs up means "yep, I got it" and generally comes from the PR author.

A "+" comes from other reviewers that agree with some feedback.

Edit:
I'm requesting this feature for PR's not necessarily for issues... but I'm hoping that they share the same set of emojis.
